src/intervals/rounding.jl

```diff
@@ -307,33 +308,32 @@ function _setrounding(::Type{Interval}, rounding_type::Symbol)
 
     end
 
-    current_rounding_type[] = rounding_type
+    # current_rounding_type[] = rounding_type
 end
 
 """
-    setrounding(Interval, rounding_type::Symbol)
+    set_directed_rounding(rounding_type::Symbol)
```

Comment:
This is a simple reminder to add the proper docs to https://juliaintervals.github.io/pages/tutorials/tutorialArithmetic/index.html#advanced_options_precision_and_rounding_modes